src/pages/About.js

```diff
@@ -1,34 +1,71 @@
-      Duis ipsum nisl, efficitur eu felis ac, tristique imperdiet turpis.
-      Curabitur quis molestie nulla. Vestibulum ante ipsum primis in faucibus
-      orci luctus et ultrices posuere cubilia curae; Nunc tempus ex lorem, a
-      tempor libero molestie a. Pellentesque in tortor non enim imperdiet
-      venenatis eget id mi. Mauris fringilla est justo, quis vehicula ipsum
-      faucibus et. Aliquam vitae nunc id est varius porttitor nec eget tellus.
-      Donec quis dictum risus, quis ornare libero. Sed erat velit, suscipit et
-      iaculis ut, tempus nec nisl. Praesent erat orci, accumsan ut risus nec,
-      egestas venenatis nunc. Integer maximus ligula commodo, fringilla risus
-      id, tempus dolor.
-    </div>
+    <Grid container className={classesCommon.pageFlexColumnLayout}>
+      <h1>O Nas</h1>
+      <Grid item>
+        <p>
+          Bardzo istotnym dla Beauty Clinic jest propagowanie zabiegów
+          anti-aging, zapobiegających procesom starzenia. W sposób skuteczny
+          usuwamy też oznaki starzenia i niedoskonałości na naszej twarzy i
+          ciele, wspomagając się nowoczesnymi technologiami. Najnowsze
+          technologie nr 1 na świecie, posiadające międzynarodowe certyfikaty
+          medyczne zapewniają skuteczne zabiegi. Profesjonalny Zespół lekarzy,
+          licencjonowanych kosmetologów i fizjoterapeutów zapewnia naturalność i
+          bezpieczeństwo.
+        </p>
+        <p>
+          Oczyszczamy, pielęgnujemy i odmładzamy, ale zanim to nastąpi zawsze
+          konsultujemy, rozmawiamy i diagnozujemy komputerowo, tak by każdy
+          zabieg był skrojony na miarę i dopasowany indywidualnie do potrzeb
+          Twojej skóry. W naszej ofercie znajdziesz szeroki wachlarz zabiegów,
+          tak by sprostać nawet najbardziej wymagającym potrzebom skóry. Prócz
+          zabiegów zawsze planujemy także pielęgnację domową dopasowaną do stylu
+          życia, możliwości i typu cery.
+        </p>
+        <p>
+          Uwielbiamy rozwijać nasze umiejętności i ciągle się szkolimy o czym
+          świadczy mnóstwo certyfikatów zdobiących nasze ściany. Wierzymy, że
+          aby zachować młodość prócz pielęgnacji cery należy wciąż się rozwijać
+          i ćwiczyć…umysł! Dlatego gdy pojawia się jakaś nowość my już się
+          szkolimy. Równie mocno co uczyć się lubimy też dzielić się naszą
+          wiedzą, dlatego zawsze możecie liczyć na nasze doradztwo, konsultacje
+          i wsparcie w dążeniu do upragnionej sylwetki czy pięknej cery.
+        </p>
+      </Grid>
+      <Grid item className={classesAbout.addressAndSocialArea}>
+        <address className={classesAbout.address}>
+          <h3 style={{ marginTop: 0 }}>Beauty Clinic</h3>
+          <p>ul.Przypadkowa 1/2</p>
+          <p>30-612 Kraków</p>
+          <p>+12 634 66 12 </p>
+        </address>
+        <div className={classesAbout.socialItem}>
```

Comment:
social item could be separate component, and whole social box would be mapped from some config array - easier to edit/add new elements and don't repeat code